src/demo/index.php

```diff
@@ -64,6 +64,9 @@ function gtag() {
                 </div>
                 <input class="param" type="text" id="font" name="font" alt="Font name" placeholder="Fira Code" value="Fira Code" pattern="^[A-Za-z0-9\- ]*$" title="Font from Google Fonts. Only letters, numbers, and spaces.">
 
+                <label for="weight">Font weight</label>
+                <input class="param" type="number" id="weight" name="weight" alt="Font weight" placeholder="400" value="400">
```

Comment:
I think it might be nice to include a min, max, and step amount to make it easier to control the value.

If you agree, feel free to use the "commit suggestion" button to apply the change.

```suggestion
                <input class="param" type="number" id="weight" name="weight" alt="Font weight" placeholder="400" value="400" min="100" max="900" step="100">
```